packages/sampler/stories/qa/Scroller.js

```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
 	}
 }
 
+class ScrollerWithSpotlightTargetCalculation extends React.Component {
```

Comment:
It's an overkill to create as a separate class as it doesn't require any react lifecycle methods or state to manage. Let's move the render part to the `.add()` of stories.